src/amo/components/Page/index.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+import makeClassName from 'classnames';
+import config from 'config';
+import * as React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import AppBanner from 'amo/components/AppBanner';
+import Header from 'amo/components/Header';
+import Card from 'ui/components/Card';
+import type { ReactRouterLocationType } from 'core/types/router';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  className?: string,
```

Comment:
I believe this should be named `componentClassName` because we usually use `className` for the top-level component that is rendered.

That makes me think that `componentProps` is not very explicit, although I get why it is called like this. Maybe we could find a better name. I suggest: `content`, so we could have `contentClassName`, `contentProps`, and maybe `useCardForContent`.